prow/cmd/deck/static/common/rerun.ts

```diff
@@ -0,0 +1,151 @@
+                <i class='material-icons state triggered' style='color: gray'>content_copy</i>
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    `;
+
+    const rerunLatestOption = rerunElement.querySelector('#rerunLatestOption');
+    const rerunCommand = rerunElement.querySelector('.rerunModal-command');
+    const rerunCommandDescription = rerunElement.querySelector('.rerunModal-commandDescription');
+    rerunLatestOption.addEventListener('click', () => {
+      commandURL = latestURL;
+      rerunCommandDescription.innerHTML = getCommandDescription(LATEST);
+      rerunCommand.innerHTML = getCommand(commandURL);
+    });
+    const rerunOriginalOption = rerunElement.querySelector('#rerunOriginalOption');
+    rerunOriginalOption.addEventListener('click', () => {
+      commandURL = originalURL;
+      rerunCommandDescription.innerHTML = getCommandDescription(ORIGINAL);
+      rerunCommand.innerHTML = getCommand(commandURL);
+    });
+    const rerunCopyButton = rerunElement.querySelector('.rerunModal-copyButton');
+    rerunCopyButton.addEventListener('click', () => {
+      copyToClipboardWithToast(getCommand(commandURL));
+    })
+    const accordion = rerunElement.querySelector('.rerunModal-accordionButton');
+    const accordionPanel = rerunElement.querySelector('.rerunModal-accordionPanel');
+    const expandIcon = rerunElement.querySelector('.rerunModal-expandIcon');
+    accordion.addEventListener('click', () => {
+      if (!accordionPanel.classList.contains('rerunModal-accordionPanel--expanded')) {
+        accordionPanel.classList.add('rerunModal-accordionPanel--expanded');
+        expandIcon.classList.add('rerunModal-expandIcon--expanded');
+      } else {
+        accordionPanel.classList.remove('rerunModal-accordionPanel--expanded');
+        expandIcon.classList.remove('rerunModal-expandIcon--expanded');
+      }
+    });
+
+    if (rerunCreatesJob) {
+      const runButton = document.createElement('a');
+      runButton.innerHTML = "<button class='mdl-button mdl-js-button mdl-button--raised mdl-button--colored'>Rerun</button>";
+      runButton.onclick = async () => {
+        gtag("event", "rerun", {
+          event_category: "engagement",
+          transport_type: "beacon",
+        });
+        const result = await fetch(commandURL, {
+          headers: {
+            "Content-type": "application/x-www-form-urlencoded; charset=UTF-8",
+            "X-CSRF-Token": csrfToken,
+          },
+          method: 'post',
+        });
+        const data = await result.text();
+        if (result.status === 401) {
+          window.location.href = window.location.origin + `/github-login?dest=${relativeURL({rerun: "gh_redirect"})}`;
+        } else if (result.status >= 400) {
+          alertToast(data);
+        } else {
+          setToast(data);
+        }
+      };
+      rerunElement.appendChild(runButton);
+    }
+  };
+
+  return i;
+}
```

Comment:
missing empty line at the bottom of the file

Reply:
fixed!